src/Compilers/Core/Portable/Emit/EditAndContinue/DefinitionMap.cs

```diff
@@ -233,12 +233,17 @@ internal VariableSlotAllocator TryCreateVariableSlotAllocator(EmitBaseline basel
                 }
                 catch (InvalidDataException)
                 {
-                    // TODO: localize message & use better error code (https://github.com/dotnet/roslyn/issues/11512): 
+                    string message = string.Format(CodeAnalysisResources.AbsolutePathExpected,
+                        MessageProvider.GetErrorDisplayString(method),
+                        MetadataTokens.GetToken(previousHandle).ToString("X8"),
+                        MessageProvider.GetErrorDisplayString(method.ContainingAssembly));
+
+                    //TODO: use better error code (https://github.com/dotnet/roslyn/issues/11512)
```

Comment:
@tmat Should a new error code be made for this error, or could we use an existing one?